app/invocation/invocation_details_card.tsx

```diff
@@ -215,15 +215,18 @@ export default class ArtifactsCardComponent extends React.Component<Props, State
                       className="copy-icon"
                       onClick={this.handleCopyClicked.bind(
                         this,
-                        `bazel ${this.props.model.started?.command} ${
-                          this.props.model.expanded?.id?.pattern?.pattern
-                        } ${this.props.model.optionsParsed?.explicitCmdLine.join(" ")}`
+                        `bazel ${
+                          this.props.model.started?.command
```

Comment:
could we just have a "formatCommand" func that returned the pretty and correct version?

Reply:
Pulled out the common bits into a function